server/services/involvedStaffService.ts

```diff
@@ -97,7 +97,48 @@ export class InvolvedStaffService {
     })
   }
 
-  public async removeInvolvedStaff(
```

Comment:
removeInvolvedStaff() is called when FEATURE_FLAG_REPORT_EDITING_ENABLED = false